cpp/util/VectorMath.h

```diff
@@ -45,7 +45,9 @@ template<class Real> struct vec3
     //! Swap with another vector
     void swap(vec3<Real>& v)
     {
-        Real tx, ty, tz;
+        Real tx;
+        Real ty;
+        Real tz;
         tx = v.x;
         ty = v.y;
         tz = v.z;
```

Comment:
Alternatively, just replace with `std::swap`. I'm just noting refactors as I read this -- feel free to leave it unresolved and it can be handled in a subsequent PR.
```suggestion
        Real tx(v.x);
        Real ty(v.y);
        Real tz(v.z);
```